lib/lumberjack/client.rb

```diff
@@ -114,18 +109,28 @@ def write(msg)
     end
 
     public
-    def write_hash(hash)
-      frame = Encoder.to_compressed_frame(hash, inc)
-      ack if unacked_sequence_size >= @window_size
-      write frame
+    def write_hash(elements)
```

Comment:
if write_hash will now accept an array, I think we should change the method name. Not sure what to  -

write_data? write_elements?